Common/src/CConfig.cpp

```diff
@@ -6838,7 +6886,23 @@ void CConfig::SetOutput(SU2_COMPONENT val_software, unsigned short val_izone) {
 
   if (val_software == SU2_COMPONENT::SU2_CFD) {
 
-    cout << "Writing solution files every " << VolumeWrtFreq <<" iterations."<< endl;
+    cout << "File writing frequency: " << endl;
+    PrintingToolbox::CTablePrinter FileFreqTable(&std::cout);
+    FileFreqTable.AddColumn("File",         25);
+    FileFreqTable.AddColumn("Frequency",    10);
+    FileFreqTable.SetAlign(PrintingToolbox::CTablePrinter::RIGHT);
+    FileFreqTable.PrintHeader();
+
+    for (auto iFreq = 0; iFreq < nVolumeOutputFrequencies; iFreq++){
+      /*--- find the key belonging to the value in the map---*/
+      for (auto& it : Output_Map) {
+        if (it.second == VolumeOutputFiles[iFreq]) 
+          FileFreqTable << it.first << VolumeOutputFrequencies[iFreq];
+      }
+    }
+
+    FileFreqTable.PrintFooter();
+
```

Comment:
I've added this to the screen output. We could also add a generic "find key based on value" function for this.